src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/layout.html.twig

```diff
@@ -88,56 +87,81 @@
                         </dl>
                     </div>
                 </div>
-            {% endif %}
-        {% endblock %}
-    </div>
 
-    <div id="content" class="container">
-        <div id="main">
-            <div id="sidebar">
-                <div id="sidebar-shortcuts">
-                    <div class="shortcuts">
-                        <a href="#" id="sidebarShortcutsMenu" class="visible-small">
```

Comment:
Removing this breaks the `Sfjs.addEventListener` call there: https://github.com/symfony/symfony/blob/f531ed2ac26886d780e7c2bef84c0c8a2dbab0ff/src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/layout.html.twig#L171